js/includes/umbrellaNav.js

```diff
@@ -0,0 +1,27 @@
+/* UMBRELLA NAV
+ * An umbrella nav element that appears at the top of every page
+ *
+*/
+var umbrellaNav = `
+	<nav class="umbrella-nav">
+		<div class="container">
+			<ul 
+				class="umbrella-nav__list"
+				aria-label="Umbrella Nav"
+			>
+				<li class="umbrella-nav__item"><a class="umbrella-nav__link" href="http://theeeventscalendar.com">Home</a></li>
```

Comment:
Luverly, except typo in the URL so it won't go where you expect it to go :-)

Also, let's switch to HTTPS.

```suggestion
				<li class="umbrella-nav__item"><a class="umbrella-nav__link" href="https://theeventscalendar.com">Home</a></li>
```